octopose/local_deploy.py

```diff
@@ -54,21 +54,43 @@ def deploy(self, data):
 
         for key, value in data['Projects'].items():
             project_name = key
-            print(project_name)
             proj_id = octo.get_project_id(project_name)
+
             if 'Version' not in value:
                 latest_release = octo.get_latest_release(proj_id)
-                version = latest_release['Version']
+                release_version = latest_release['Version']
             else:
-                version = value['Version']
+                release_version = value['Version']
+            print("{0} - {1}".format(project_name, release_version))
```

Comment:
personally would add a blank line above this and remove the one below ... but meh whatever you like ;)

Reply:
Also might be an idea to move to structured logging at some point, this looks nice http://www.structlog.org/en/stable/ but maybe not for the size of this project.